Refactor Acct Preferences (#2451)
getsentry/sentry-docs: src/docs/product/accounts/user-settings/index.mdx

  - /workflow/user-settings/
---

> [Account preferences](https://sentry.io/settings/account/details/) provide customization of your Sentry experience like controling the frequency of your [notifications](#notifications) and [changing your primary email](#emails). Manage your account by selecting "User settings" from the dropdown under your organization’s name.

Reviewer: ```suggestion
[Account preferences](https://sentry.io/settings/account/details/) help you customize your Sentry experience. Manage your account by selecting "User settings" from the dropdown under your organization’s name. On this page, you can control the frequency of your [email notifications](#notifications) and [change your primary email](#emails) as well as update your security settings. 
```

## Security Settings

Security Settings include options to reset a password, sign out of all devices, and enable two-factor authentication.

### Two-Factor Authentication

If you cannot receive two-factor authentication codes, such as if you lose your device, use recovery codes to access your account. Click "View Codes" to download, print, or copy your codes to a secure location.

<Note>
<markdown>
Clicking "Sign out of all devices" will end your sessions with any device logged in to Sentry using your account.
</markdown>
</Note>

## Notifications

> Notification emails send when an issue state changes, a release deploys, or there's a spike in your quota. Fine-tune notifications per project by navigating to [Workflow Notifications](https://sentry.io/settings/account/notifications/workflow/). For more details about Notifications, see our [full documentation on Alerts & Notifications](/product/alerts-notifications/).

Reviewer: ```suggestion
Sentry emails you a notification when an issue state changes, a release deploys, or there's a spike in your quota. Fine-tune notifications per project by navigating to [Workflow Notifications](https://sentry.io/settings/account/notifications/workflow/). For more details about Notifications, see our [full documentation on Alerts & Notifications](/product/alerts-notifications/).
```

### Email Routing

> [Email Routing](https://sentry.io/settings/account/notifications/email/) controls the emails sent for each project. These notifications default to the email address used to set up your Sentry account. Route email notifications to an alternative email address to organize your project notifications. For example, infrastructure-related projects can route to an email dedicated to your team's infrastructure.

Reviewer: ```suggestion
[Email Routing](https://sentry.io/settings/account/notifications/email/) controls the email to which notifications are sent for each project. These notifications default to the email address provided when you set up your Sentry account. You can route emails to an alternative email address when, for example, you want to notify only the team associated with a specific project.
```

### Weekly Reports

> Sentry generates [Weekly Reports per project](https://sentry.io/settings/account/notifications/reports/) and sends these reports once a week on Mondays. The following report sample shows details on Events Seen This Week, Events by Issue Type, Events by Project, and activity over time.

Reviewer: Suggest we modify the content here to make it slightly more evergreen should our reports change (so we'd only need to swap out the shot)
```suggestion
Sentry generates [Weekly Reports per project](https://sentry.io/settings/account/notifications/reports/) and sends these reports once a week on Mondays. For example:
```

![Weekly Report email sample](us-ereport.png)

## Emails

> The email address used to log into your Sentry account is, by default, your primary email address. Add an alternative email address in the [Add Secondary Emails](https://sentry.io/settings/account/emails/) section. After verifying your secondary email, you can set it as the primary.

Reviewer: ```suggestion
The email address used to log into your Sentry account is, by default, your primary email address. Add an alternative email address in the [Add Secondary Emails](https://sentry.io/settings/account/emails/) section. After verifying your secondary email, you can set it to become your primary email.
```
